README.md

```diff
@@ -1,2 +1,123 @@
+
+### 🛠 Setup <a name="setup"></a>
+
+1. Clone the project repository:
+
+   ```bash
+   git clone https://github.com/PabloBona/recipe-app.git
+    ```
+2. Navigate to the project folder:
+
+   ```bash
+   cd recipe-app
+   ```
+### Prerequisites <a name="prerequisites"></a>
+Before you can run the Blog App, make sure you have the following prerequisites and configurations in place:
+
+- **Ruby**: Ensure that Ruby is installed on your machine. You can check if it's installed by running `ruby -v` in your terminal. If it's not installed, you can download and install it from the [official Ruby website](https://www.ruby-lang.org/en/documentation/installation/).
+
+- **Ruby on Rails**: You'll need the Ruby on Rails framework installed. If you don't have it yet, you can install it using the following command:
+  ```
+  gem install rails
+  ```
+
+- **Code Editor**: You should have a code editor like Visual Studio Code (VSCode) installed. You can download VSCode from the [official website](https://code.visualstudio.com/).
+
+- **Git**: Git is essential for version control. Make sure you have Git installed and configured on your system. You can download Git from the [official Git website](https://git-scm.com/downloads).
+
+- **PostgreSQL**: Ensure that PostgreSQL is installed and properly configured on the server where you are developing the project. This project assumes the use of PostgreSQL as the default database. You can download PostgreSQL from the [official PostgreSQL website](https://www.postgresql.org/download/).
+
+- **Ruby on Rails Configuration**: Make sure your Ruby on Rails project is configured to use PostgreSQL as the default database.
+
+With these prerequisites and configurations in place, you'll be ready to set up and run the Blog App.
+
+### 📖 Usage <a name="usage"></a>
+Follow these steps to use the Blog App:
+
+Open your terminal.
+
+Navigate to the directory where the project files are located:
+
+Copy code
+```bash
+cd /path/to/your/project/files
+```
+Start the Rails server:
+
+Copy code
+```bash
+rails server
+```
+Open your web browser and go to http://localhost:3000 to access the Blog App.
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Andrea Manuel**
+
+- GitHub: [@AndreaM2429](https://github.com/AndreaM2429)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/andreamanuel24/)
+
+👤 Pablo Bonasera
+- Github: [@PabloBona](https://github.com/PabloBona)
+- Linkedin: [Pablo Bonasera](https://www.linkedin.com/in/pablo-bonasera/)
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome! 
+Feel free to check the issues [page](https://github.com/PabloBona/recipe-app) and contribute to the project.
+
+## ⭐️ Show Your Support <a name="support"></a>
+If you find this project helpful, consider giving it a ⭐️ to show your support!
+
+## 📜 License <a name="license"></a>
+This project is [MIT](https://github.com/PabloBona/Blog-app/recipe-app/dev/MIT.md) licensed.
+
```

Comment:
- [ ] This link goes to a 404 page, I will encourage you to fix it.

Reply:
Hi @Olamarx, done.
Thanks for the observation.

Reply:
This change has been done well